### PR TITLE
[docs] Move settings related to emailing users to settings page

### PR DESCRIPTION
**Pull request by EdgeKing810**
Also, I tried improving the english in some other docs files.
GCI Task: https://codein.withgoogle.com/dashboard/task-instances/5417598670864384/

### CODE REVIEW

File: docs/source/general/settings.rst [hunk 1]
```diff
@@ -151,9 +151,26 @@ with other freeradius modules.
...
 
-    * `DJANGO_FREERADIUS_BATCH_MAIL_SUBJECT <importing_users.html#django-freeradius-batch-mail-subject>`_
-    * `DJANGO_FREERADIUS_BATCH_MAIL_MESSAGE <importing_users.html#django-freeradius-batch-mail-message>`_
-    * `DJANGO_FREERADIUS_BATCH_MAIL_SENDER <importing_users.html#django-freeradius-batch-mail-sender>`_
+Emails can be sent to users whose usernames or passwords have been autogenerated. The content of these emails are customisable. Many settings have been defined for doing that:
+
+``DJANGO_FREERADIUS_BATCH_MAIL_SUBJECT``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It is the subject of the mail to be sent to the users. Eg: ``Login Credentials``
```
Comment (nemesisdesign) [inline docs/source/general/settings.rst]: specify the default value as it's done for the other settings (find it here: https://github.com/openwisp/django-freeradius/blob/master/django_freeradius/settings.py#L23-L24)

File: docs/source/general/settings.rst [hunk 2]
```diff
@@ -151,9 +151,26 @@ with other freeradius modules.
...
+
+``DJANGO_FREERADIUS_BATCH_MAIL_MESSAGE``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The message should be a string in the format ``Your username is {} and password is {}``.
```
Comment (nemesisdesign) [inline docs/source/general/settings.rst]: specify the default value as it's done for the other settings (find it here: https://github.com/openwisp/django-freeradius/blob/master/django_freeradius/settings.py#L23-L24)

File: docs/source/general/settings.rst [hunk 3]
```diff
@@ -151,9 +151,30 @@ with other freeradius modules.
...
 
-    * `DJANGO_FREERADIUS_BATCH_MAIL_SUBJECT <importing_users.html#django-freeradius-batch-mail-subject>`_
-    * `DJANGO_FREERADIUS_BATCH_MAIL_MESSAGE <importing_users.html#django-freeradius-batch-mail-message>`_
-    * `DJANGO_FREERADIUS_BATCH_MAIL_SENDER <importing_users.html#django-freeradius-batch-mail-sender>`_
+Emails can be sent to users whose usernames or passwords have been autogenerated. The content of these emails can be customized with the settings explained below.
+
+``DJANGO_FREERADIUS_BATCH_MAIL_SUBJECT``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Default**: ``""``
```
Comment (nemesisdesign) [inline docs/source/general/settings.rst]: I really don't understand why you are putting this default here, have you looked at the default in the code?
https://github.com/openwisp/django-freeradius/blob/master/django_freeradius/settings.py#L23-L24

File: docs/source/general/settings.rst [hunk 4]
```diff
@@ -151,9 +151,30 @@ with other freeradius modules.
...
+
+It is the subject of the mail to be sent to the users. Eg: ``Login Credentials``.
+
+``DJANGO_FREERADIUS_BATCH_MAIL_MESSAGE``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Default**: ``""``
```
Comment (nemesisdesign) [inline docs/source/general/settings.rst]: same here

File: docs/source/general/settings.rst [hunk 5]
```diff
@@ -151,9 +151,30 @@ with other freeradius modules.
...
+
+The message should be a string in the format ``Your username is {} and password is {}``.
+
+The text could be anything but should have the format string operator ``{}`` for ``.format`` operations to work.
+
+``DJANGO_FREERADIUS_BATCH_MAIL_SENDER``
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It is the sender email which is also to be configured in the SMTP settings.
```
Comment (nemesisdesign) [inline docs/source/general/settings.rst]: specify the default value here as well (the actual default mentioned in the code), why should this setting be different from the others?

**Comment by EdgeKing810** [comment]
Sorry about all that... Blame it on my assumptions!

**Pull request merged**